modules/flowable-engine/src/main/java/org/flowable/engine/impl/dynamic/AbstractDynamicStateManager.java

```diff
@@ -892,7 +893,27 @@ protected ExecutionEntity migrateExecutionEntity(ExecutionEntity parentExecution
                     .executionId(childExecution.getId()).singleResult();
             task.setProcessDefinitionId(childExecution.getProcessDefinitionId());
             task.setTaskDefinitionKey(newFlowElement.getId());
-            task.setName(newFlowElement.getName());
+
+            // Set name
+            String name = null;
+            if(newFlowElement.getName() != null) {
```

Comment:
small nitpick: there should be a space in `if(` like other if statements.   See line 909 also.